src/application/services/useNote.ts

```diff
@@ -158,6 +158,13 @@ export default function (options: UseNoteComposableOptions): UseNoteComposableSt
    */
   const parentNote = ref<Note | undefined>(undefined);
 
+  /**
+   * Note parents of the actual note
+   *
+   * Actual note by default
+   */
+  const noteParents = ref<Note[]>([]);
```

Comment:
i think that making `noteParents` reactive is a bad thing to do
We can update them only by request to api `/GET note` so i can't see the case, when we update noteParents and imidiately change displayed content somewhere